sdk/agrifood/Azure.Verticals.AgriFood.Farming/src/Generated/Docs/ApplicationData.xml

```diff
@@ -8,7 +8,7 @@ This sample shows how to call GetCascadeDeleteJobDetailsAsync with required para
 var credential = new DefaultAzureCredential();
 var client = new FarmBeatsClient(credential).GetApplicationDataClient("2022-11-01-preview");
 
-Response response = await client.GetCascadeDeleteJobDetailsAsync("<jobId>");
+Response response = await client.GetCascadeDeleteJobDetailsAsync("<jobId>", new RequestContext());
```

Comment:
Should we omit this in examples? I once saw a comment in the example composer saying that we should omit